styles/themes/maple.ts

```diff
@@ -0,0 +1,57 @@
+import { css } from 'styled-components';
+
+export const maple = css`
+    --maple_01: rgb(222, 121, 146);
```

Comment:
add and use the contituency dashboard theme here, then change them to district when you clone it

Reply:
working on that!